mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -144,10 +149,11 @@ class CompletionProvider(
                 item.setAdditionalTextEdits(edits.asJava)
                 item.setTextEdit(textEdit(short + suffix))
             }
-          case _ =>
+          case _ if importPosition.isEmpty =>
```

Comment:
I'm not entirely sure this is accurately coupled with "the current position is an import"... could there be any other case in which importPosition is empty?

Reply:
Judging by the implementation of `autoImportPosition` it only returns `None` for import expressions, but there's nothing in the contract of that method which prevents this implementation detail from changing in the future. It would be better do something custom instead which implements exactly what we want for this purpose https://gist.github.com/olafurpg/d15833f9b519ac286e36a5daad8f07a5

Reply:
I'll give this a try :)